doc/encryption.md

```diff
@@ -106,18 +105,16 @@ protected void onCreate(Bundle savedInstanceState) {
  
     SQLiteDatabase.loadLibs(this);
  
-    // Create a DatastoreManager with encryption using 
+    // Get a DocumentStore instance with encryption using 
     // application internal storage path and a key
-    File path = getApplicationContext().getDir("datastores", MODE_PRIVATE);
+    File path = getApplicationContext().getDir("documentstores", MODE_PRIVATE);
     KeyProvider keyProvider = 
             new SimpleKeyProvider("testAKeyPasswordtestAKeyPassword".getBytes());
  
-    DatastoreManager manager = DatastoreManager.getInstance(path.getAbsolutePath());
- 
-    Datastore ds = null;
+    DocumentStore ds = null;
     try {
-        ds = manager.openDatastore("my_datastore", keyProvider);
-    } catch (DatastoreNotCreatedException e) {
+        ds = DocumentStore.getInstance(new File(path, "my_documentstore"), keyProvider);
+    } catch (DocumentStoreNotOpenedException e) {
         e.printStackTrace();
     }
         
```

Comment:
Update exception at line 144 (https://github.com/cloudant/sync-android/pull/467/files?diff=split#diff-0ea0055410d32eaef0440d8985d51bbdR144) to include `com.cloudant.sync.documentstore`?